dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -236,6 +234,8 @@
  */
 public class ESContentletAPIImpl implements ContentletAPI {
 
+    private static Lazy<Boolean> ENABLED_UNIQUE_FIELDS_DATA_BASE_VALIDATION = Lazy.of(() ->
+            Config.getBooleanProperty("ENABLED_UNIQUE_FIELDS_DATA_BASE_VALIDATION", false));
```

Comment:
Rename to FEATURE_FLAG_DB_UNIQUE_FIELD_VALIDATION

We always default to true. I will confirm with you if we default to this new DB implementation or not. @freddyDOTCMS

Reply:
I have a card to enabled it, here https://github.com/dotCMS/core/issues/30306